src/starkware/starknet/core/os/contract_hash.py

```diff
@@ -20,19 +20,21 @@
 
 CAIRO_FILE = os.path.join(os.path.dirname(__file__), "contracts.cairo")
 
-
 def load_program() -> Program:
     return compile_cairo_files(
         [CAIRO_FILE],
         prime=DEFAULT_PRIME,
         main_scope=ScopedName.from_string("starkware.starknet.core.os.contracts"),
     )
 
+# Load the Cairo Program as a static so that it gets initialized at the
+# beginning of a script run.
+CAIRO_PROGRAM = load_program()
```

Comment:
Same issue as above, even if you add the caching, you want this function to run as early as possible while other things are running in the background so that you don't block on it while the tests run